thinc_apple_ops/ops.pyx

```diff
@@ -7,16 +7,22 @@ from . import blas
 from .blas cimport saxpy, sgemm
 
 
+def init_cblas() -> CBlas:
+    cdef CBlas cblas = CBlas()
+    cblas.set_saxpy(saxpy)
+    cblas.set_sgemm(sgemm)
+    return cblas
+
+cblas = init_cblas()
```

Comment:
I think the overhead of initializing `CBlas` is really small and this can introduce a lot of potential problems? E.g. if a downstream user decides to call `set_*` on the returned object, it changes the object for everyone.

Reply:
I also created a global in `numpy_ops`, we also probably shouldn't do that 😨 .